atest/testdata/running/continue_on_failure_tag.robot

```diff
@@ -199,16 +197,16 @@ Recursive Continue in test without tag and two nested UK with and without recurs
     Failure in user keyword with recursive tag     run_kw=Failure in user keyword without tag
     Fail   This should not be executed
 
-Recursive Continue in test without tag and two nested UK without and with recursive tag
+No recursive continue in user keyword
```

Comment:
Is this name accurate? Doesn't the test validate recursive continue works also with nested keywords?